js/quiz.js

```diff
@@ -471,12 +476,22 @@
         if (correctCount < 3) {
           $('#quiz .finished').addClass('can-be-better')
           $('#quiz .finished').removeClass('almost-all-right')
+
+          gtag("event", "quiz_onAnsweredLessThan2Questions", {
+            event_category: "quiz_interaction",
+          });
```

Comment:
```suggestion
gtag('event', 'quiz_finish', {
  'event_category': 'quiz_interaction',
  'event_label': 'quiz_finish',
  'value': 'correctCount'
});
```

במקום לעשות איוונטים לפי החלוקה השרירותית שלפיה רושמים הודעות למשתמש (ענה נכון פחות מ2, ענה נכון 3 או 4, ענה נכון על הכל), בוא נעשה אירוע אחיד וניתן לו value שונה.

Reply:
רעיון הרבה יותר טוב.